web/frontend/src/components/slam/index.svelte

```diff
@@ -350,12 +350,18 @@ const handleStartMapping = async () => {
       mapNameError = 'Please enter a name for this map';
       return;
     }
+    console.log("In handleStartMapping, mappingDetails: ", overrides.mappingDetails)
```

Comment:
Can we remove these console logs?

Reply:
oh, it's not ready for review yet; these are there on purpose since I didn't have a chance to test this yet.

Reply:
I will remove all logs after I run more tests in staging.

Reply:
Done.